code/game/machinery/recycle_vendor.dm

```diff
@@ -636,6 +688,19 @@
 		if(materials_stored[matname] > sellthreshold)
 			sellonething(matname, materials_stored[matname] - sellthreshold) // sell excess
 		
+/obj/machinery/amesilo/proc/updatesubsidy()
+	if(my_account.money <= 400)
+		if(department_accounts[DEPARTMENT_COMMAND])
+			var/datum/money_account/commandaccount = department_accounts[DEPARTMENT_COMMAND]
+			if(commandaccount.money < MINIMUM_BUDGET)
+				return FALSE
+			var/totransfer = MINIMUM_BUDGET-my_account.money
+			taxdebt += totransfer
```

Comment:
the subsidy increases tax debt by the amount which is given to the AME's account, ensuring it cannot be used as profits.